.circleci/config.yml

```diff
@@ -61,6 +61,32 @@ jobs:
       - image: circleci/ruby:2.6
     <<: *rubocop
 
+  edge-rubocop:
+    docker:
+      - image: circleci/ruby
+    steps:
+      - checkout
+      - run:
+          name: Use latest RuboCop from `master`
+          command: |
+            echo "gem 'rubocop', github: 'rubocop-hq/rubocop'" > Gemfile.local
+      - run: bundle install --no-cache
+      - run: rake spec
+      - run: rake internal_investigation
+
+  earliest-supported-rubocop:
+    docker:
+      - image: circleci/ruby
+    steps:
+      - checkout
+      - run:
+          name: Use earliest supported RuboCop
+          command: |
+            echo "gem 'rubocop', '0.68.1'" > Gemfile.local
```

Comment:
Our gemspec says `spec.add_runtime_dependency 'rubocop', '>= 0.60.0'`, so I would imaging we should install `0.60.0` here. Where does the `0.68.1` come from?

Reply:
Ah, from #804 probably.

It would be nice if we could write this version number dynamically, e.g.:
```
ruby -e "version = File.read('rubocop-rspec.gemspec')[/^  spec\.add_runtime_dependency 'rubocop', '>= (\d+\.\d+\.\d+)'$/, 1]; File.write('Gemfile.local', %(gem 'rubocop', '#{version}'))"
```